.github/workflows/opened-pr-test.yml

```
@@ -12,7 +12,97 @@ jobs:
+
+      mongodb:
+        image: mongo:latest
+        env:
+          MONGO_INITDB_ROOT_USERNAME: ${{ secrets.MONGO_ROOT_USERNAME }}
+          MONGO_INITDB_ROOT_PASSWORD: ${{ secrets.MONGO_ROOT_PASSWORD }}
+        ports:
+          - 27017:27017
+
     steps:
+      - name: Wait for Elasticsearch
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2215:warning:8:1: This flag is used as a command name. Bad line break or missing [ .. ]?

(shellcheck)